server/errors_test.go

```diff
@@ -0,0 +1,75 @@
+// Copyright 2012-2021 The NATS Authors
```

Comment:
Since this is a new file, data should simply be `2020`.

Reply:
fixed